doc/reference/reference_lua/net_box.rst

```diff
@@ -163,54 +163,91 @@ Below is a list of all ``net.box`` functions.
     such as select, update or delete.
     
     Possible options:
+    
+    *   ``user/password``: you have two ways to connect to a remote host: 
+        using :ref:`URI <index-uri>` or using the options `user` and `password`. For 
+         example, instead of ``connect('username:userpassword@localhost:33301')`` 
+         you can write ``connect('localhost:33301', {user = 'username', password='userpassword'})``.
```

Comment:
```suggestion
    
    *   ``user/password``: you have two ways to connect to a remote host: 
        using :ref:`URI <index-uri>` or using the options `user` and `password`. For 
        example, instead of ``connect('username:userpassword@localhost:33301')`` 
        you can write ``connect('localhost:33301', {user = 'username', password='userpassword'})``.
```
There are extra spaces at the beginning of the last two lines. This makes Sphinx interpret the paragraph as a so-called "definition list" – as though there is a term in bold and an explanatory text following it:

![image](https://user-images.githubusercontent.com/40490252/171824065-99ff2f92-7176-47da-9b08-a4ce23a981bf.png)

Reply:
Fixed.